src/Compute/Compute/help/Set-AzVMSqlServerExtension.md

```diff
@@ -28,42 +28,42 @@ The **Set-AzVMSqlServerExtension** cmdlet sets the AzureSQL Server extension on
 ### Example 1: Set automatic patching settings on a virtual machine
 ```powershell
 $AutoPatchingConfig = New-AzVMSqlServerAutoPatchingConfig -Enable -DayOfWeek "Thursday" -MaintenanceWindowStartingHour 11 -MaintenanceWindowDuration 120 -PatchCategory "Important"
-Get-AzVM -ServiceName "Service02" -Name "VirtualMachine11" | Set-AzVMSqlServerExtension -AutoPatchingSettings $AutoPatchingConfig | Update-AzVM
```

Comment:
https://social.technet.microsoft.com/Forums/en-US/04ac8173-1496-467a-a54b-2c5faa331a1b/how-can-i-find-the-servicename-for-a-vm-running-in-azure 
 
I think it can be replaced by -ResourceGroupName